docs/design/rebuild_out_synced_shards.md

```diff
@@ -0,0 +1,17 @@
+# Force rebuild out-synced Shards with broken Merkle Tree
+
+## Overview
+
+TODO
+
+## How to use
```

Comment:
on main README it is mentioned that 'user is aware of the risks'. Please describe these risks here.

Reply:
will add in the next PR with updated doc